test/parallel/test-url-parse-invalid-input.js

```diff
@@ -12,9 +12,13 @@ const url = require('url');
   0.0,
   0,
   [],
-  {}
-].forEach(function(val) {
-  assert.throws(function() { url.parse(val); }, TypeError);
+  {},
+  () => {},
+  Symbol('foo')
+].forEach((val) => {
+  assert.throws(() => { url.parse(val); },
+                /^TypeError: Parameter "url" must be a string, not (undefined|boolean|number|object|function|symbol)$/);
 });
 
-assert.throws(function() { url.parse('http://%E0%A4%A@fail'); }, /^URIError: URI malformed$/);
+assert.throws(() => { url.parse('http://%E0%A4%A@fail'); },
```

Comment:
I would say the braces should *not* be omitted. That would change it from merely running `url.parse()` to returning a value. If it was `{ return url.parse(...); }` then omitting the braces (and the `return`) would make sense. It's mostly a semantic difference, but I think it's important.